README.md

```diff
@@ -49,6 +56,21 @@ make
 make test
 ```
 
+Optionally, you can install [DeepMind Lab](https://deepmind.com/blog/open-sourcing-deepmind-lab) and [Project Malmo](https://www.microsoft.com/en-us/research/project/project-malmo).
+
+To use DeepMind Lab, please follow the instructions on [https://github.com/deepmind/lab](https://github.com/deepmind/lab) and run the cmake command
```

Comment:
Why follow the instructions? We only need to tell the users to install bazel and opengl by doing sudo apt-get